tools/build-sunder-debian-packages.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+set -u
+
+
+SUNDER_CODE=/sunder
+
+if [[ ! -d "${SUNDER_CODE}" ]]; then
+    printf "${SUNDER_CODE} directory not found!\\n"
+    exit 1
+fi
+
+icns2png --extract --output "${SUNDER_CODE}/build/icons/" "${SUNDER_CODE}/build/icon.icns"
+npm install
+npm rebuild
```

Comment:
Pretty sure you don't want to call `npm rebuild` anymore, it will actually break things because it will rebuild native module for the system's version of Node rather than Electron's version of Node. The order of events is:

1. `npm install` builds native modules for system Node by default
2. There is a postinstall hook in `package.json` that runs `electron-builder install-app-deps`, which takes care of rebuilding native modules against Electron's version of Node.

I'm pretty sure calling `npm rebuild` at this point will do the wrong thing, e.g. similar to how it was causing breakage on CircleCI that I fixed in 960dc2b575aaae32a0d6bb6e534b831bc2ed24f1.

Reply:
Honestly the hardest thing about returning to work on this was grokking all of the interactions between native modules and the rest of our code/dependencies/NPM/etc. 😓